src/dotnet-uninstall/Shared/BundleInfo/Bundle.cs

```diff
@@ -2,10 +2,11 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Linq;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo.Versioning;
 
 namespace Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo
 {
-    internal abstract class Bundle : IEquatable<Bundle>
+    public abstract class Bundle : IEquatable<Bundle>
```

Comment:
It is a long story. It was because test classes have to be `public` and previously they inherited from a class that uses those classes as generic type parameters, which forced them to be `public`. Later I changed inheritance to `static` methods, so they don't need to be `public` anymore. However I forgot to change them back. Thanks very much for the nice catch! I have fixed them.